src/main/java/mate/academy/Main.java

```diff
@@ -2,11 +2,12 @@
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Main {
     public static void main(String[] args) {
         // Feel free to play with AsyncRequestProcessor in this main method if you want
-        ExecutorService executor = null; // Provide implementation that fits your needs
+        ExecutorService executor = Executors.newFixedThreadPool(4);
```

Comment:
Consider handling the potential issue where the ExecutorService may not shut down immediately when the application ends. You might want to add a shutdown hook or use try-with-resources to ensure that the executor is properly shut down.